Google.Cloud.Functions.Framework.Tests/CloudEventAdapterTest.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020, Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using CloudNative.CloudEvents;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Functions.Framework.Tests
+{
+    /// <summary>
+    /// Sample no-op test just to get the infrastructure correct.
```

Comment:
nit: I think this comment is now obsolote.

Reply:
Whoops, that's very true. Will fix, thanks!